{"name": "test_account_wrong_login", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'LoginElement' has no attribute 'password_input_box'", "trace": "self = <test_cases.test_login.test_login.TestLogin object at 0x000002260ADFA358>\nstate_login_class = <OperationalLayer.Login.login.LoginOperate object at 0x000002260ADC3DA0>\n\n    @allure.story('这是一个测试账号错误登录的case')\n    def test_account_wrong_login(self, state_login_class):\n        '''\n        用例描述：输入错误管理员账号，正确密码，验证码（随便），登录报错\n        :return:\n        '''\n        state_login_class.get_login_url()\n        state_login_class.input_account(abnormal_login_data.account, abnormal_login_data.password,\n>                                       abnormal_login_data.verification_code)\n\ntest_cases\\test_login\\test_login.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <OperationalLayer.Login.login.LoginOperate object at 0x000002260ADC3DA0>, account = 'admin1', password = 'admin123', verification_code = '1111'\n\n    @allure.step('步骤2：输入账号，密码，验证码')\n    def input_account(self, account, password, verification_code):\n        self.send_keys(LoginElement.account_input_box, account)\n>       self.send_keys(LoginElement.password_input_box, password)\nE       AttributeError: type object 'LoginElement' has no attribute 'password_input_box'\n\nOperationalLayer\\Login\\login.py:24: AttributeError"}, "description": "\n        用例描述：输入错误管理员账号，正确密码，验证码（随便），登录报错\n        :return:\n        ", "steps": [{"name": "步骤1：到达登录页面", "status": "passed", "start": 1579483514587, "stop": 1579483515006}, {"name": "步骤2：输入账号，密码，验证码", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'LoginElement' has no attribute 'password_input_box'\n", "trace": "  File \"c:\\users\\keda\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\allure_commons\\_allure.py\", line 157, in impl\n    return func(*a, **kw)\n  File \"E:\\automatic\\ui-automatic\\OperationalLayer\\Login\\login.py\", line 24, in input_account\n    self.send_keys(LoginElement.password_input_box, password)\n"}, "parameters": [{"name": "account", "value": "'admin1'"}, {"name": "password", "value": "'admin123'"}, {"name": "verification_code", "value": "'1111'"}], "start": 1579483515006, "stop": 1579483515152}], "attachments": [{"name": "stdout", "source": "f987f0f7-2d23-4976-b239-5b4e1a28b3de-attachment.txt", "type": "text/plain"}], "start": 1579483514587, "stop": 1579483515152, "uuid": "06e9db7e-3a76-4c34-b3ab-bc0832fd1787", "historyId": "b308178dd37a299f6d4684f5efc4274c", "testCaseId": "1a9d587fd15c0e192eb1d219417b90b6", "fullName": "test_cases.test_login.test_login.TestLogin#test_account_wrong_login", "labels": [{"name": "feature", "value": "登录"}, {"name": "story", "value": "这是一个测试账号错误登录的case"}, {"name": "parentSuite", "value": "test_cases.test_login"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-IJ0L9OR"}, {"name": "thread", "value": "20184-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_login.test_login"}]}